core/src/epicli/data/common/ansible/playbooks/roles/kafka/defaults/main.yml

```diff
@@ -1,5 +1,5 @@
 specification:
   kafka_var:
```

Comment:
I agree, but changing it will cause a breaking change for existing cluster configuration. It will require to add validation to yaml if those are updated. 
I propose to create a "Kafka role refactoring" task to prevent "wild" refactorings across the solution.